README.md

```diff
@@ -1 +1,13 @@
-# angular-testing-services-lab
\ No newline at end of file
+# Use ngMocks and Jasmine/Karma
+
+## Objectives
+
+- Test the created Services
+
+## Instructions
+
+In this repo we have a service named `UserService`. This has two functions.
+
+The first function is to get the user's logged in information from `/rest/user`. Test this method, using `$httpBackend` to mock the response.
```

Comment:
Does it matter what they mock the response to be, since they'll then just make sure that the method returns whatever response they've mocked out?